_clients/k8s-operator.md

```diff
@@ -0,0 +1,146 @@
+
+```yml
+apiVersion: v1
+clusters:
+- cluster:
+    certificate-authority: /Users/naarcha/.minikube/ca.crt
+    extensions:
+    - extension:
+        last-update: Mon, 29 Aug 2022 10:11:47 CDT
+        provider: minikube.sigs.k8s.io
+        version: v1.26.1
+      name: cluster_info
+    server: https://127.0.0.1:61661
+  name: minikube
+contexts:
+- context:
+    cluster: minikube
+    extensions:
+    - extension:
+        last-update: Mon, 29 Aug 2022 10:11:47 CDT
+        provider: minikube.sigs.k8s.io
+        version: v1.26.1
+      name: context_info
+    namespace: default
+    user: minikube
+  name: minikube
+current-context: minikube
+kind: Config
+preferences: {}
+users:
+- name: minikube
+  user:
+    client-certificate: /Users/naarcha/.minikube/profiles/minikube/client.crt
+    client-key: /Users/naarcha/.minikube/profiles/minikube/client.key
+```    
+   
+5. Enter `make install` to create the CustomResourceDefinition that runs in your Kubernetes cluster. 
+6. Start the OpenSearch Kubernetes Operator. Enter `make run`. 
+
+## Verify Kubernetes deployment
+
+To ensure that Kubernetes recognizes the OpenSearch Kubernetes Operator as a namespace, enter `k get ns | grep opensearch`. Both `opensearch` and `opensearch-operator-system` should appear as `Active`.
+
+With the operator active, use `k get pod -n opensearch-operator-system` to make sure that the operator's pods are running. 
+
+```
+NAME                                              READY   STATUS   RESTARTS   AGE
+opensearch-operator-controller-manager-<pod-id>   2/2     Running  0          25m
+```
+
+With the Kubernetes cluster running, you can now run OpenSearch inside the cluster.
+
+## Deploy a new OpenSearch cluster
+
+From your cloned OpenSearch Kubernetes Operator repo, navigate to the `opensearch-operator/examples` directory. There you'll find the `opensearch-cluster.yaml` file, which can be customized to the needs of your cluster, including the `clusterName` that acts as the namespace in which your new OpenSearch cluster will reside.
+
+With your cluster configured, run the `kubectl apply` command.
+
+```
+kubectl apply -f opensearch-cluster.yaml
+```
+
+The operator creates several pods, including a bootstrap pod, three OpenSearch cluster pods, and one Dashboards pod. To connect to your cluster, use the `port-forward` command.
+
+```
+kubectl port-forward svc/my-cluster-dashboards 5601
+```
+
+Open http://localhost:5601 in your preferred browser and log in with the default demo credentials `admin / admin`. You can also run curl commands against the OpenSearch REST API by forwarding to port 9200.
+
+```
+kubectl port-forward svc/my-cluster 9200
+```
+
+In order to delete the OpenSearch cluster, delete the cluster resources. The following command deletes the cluster namespace and all its resources.
+
+```
+kubectl delete -f opensearch-cluster.yaml
```

Comment:
if this also deletes the PVCs (= the data) then you should add a big fat warning as this will be a non-reversible option with data loss (which may or may not be intentional)!